components/bl-multiple-select-component/src/options/select-all-checkbox.js

```diff
@@ -0,0 +1,12 @@
+import { Checkmark } from './checkmark';
+
+export function SelectAllCheckbox({ label, isAllOptionsSelected, handleSelectAll }) {
+  return (
+    <div
+      onClick={ handleSelectAll }
+      className="select-all-checkbox">
+      <Checkmark isOptionSelected={ isAllOptionsSelected } />
```

Comment:
```
<Checkmark checked={ isAllOptionsSelected } />
```

Checkmark doesn't know about options, isn't it?

Reply:
Yeap